docs/docs/tutorials/get-started/further-readings.md

```diff
@@ -3,3 +3,32 @@ sidebar_position: 6
 ---
 
 # Further readings
+
+### [⚔️ Protect your API](../../recipes/protect-your-api/README.md)
+
+For native and single page apps, you'll need to call one or more API endpoints to retrieve and update data.<br/>
+Learn more about identifying who's who and keeping your API secure.
```

Comment:
Prefer following the GFM (GitHub Flavored Markdown) style:
use a blank line instead of `<br/>` to separate the contents

Reply:
blank line will separate into paragraphs, which is not we want (too much space)